lib/controllers/users.js

```diff
@@ -17,8 +17,8 @@ var users = module.exports = express.Router({ mergeParams: true });
 var User = mongoose.model("User");
 var Patient = mongoose.model("Patient");
 
-const makeRequest = (data, callback) => {
-  request.post(data, callback);
+const makeRequest = (data, callback, method = "post") => {
+  request[method](data, callback);
```

Comment:
This might be nitpicky: I usually don't like one line functions, unless that line is really funky or its purpose isn't obvious. In this case, I don't think
```javascript
makeRequest({ stuff }, (res) => doStuff(res), "post");
```
is as nice as just using request directly like:
```javascript
request.post({ stuff }, (res) => doStuff(res));
```

Reply:
I think ideally we'd have a `NotificationService` object with methods that return promises and use `request` under the hood, but since we aren't there yet, I'm not sure if this small abstraction is necessary

Reply:
Good point @orndorffgrant . I guess I chose that convention because it was used for all previous requests to authenticate against Auth and make request to Notification MS, hence was just following the established pattern. But I agree. It would probably be nice to have a service within Orange API that handles communication with Notification MS that wraps all call backs with promises. Will write a ticket for it.

Reply:
Thinking about it more: The Notification service (and all other services) should probably include a client service library in its repo, which can be published as an npm package, and pulled into other projects like this one

Reply:
I have had the same sentiment about this useless `makeRequest` function all over our code. @ElijahGeorge we might as well git rid of it in this file and directly use `request.get`, `request.post`, `request.whateverYouNeed`...